package-lock.json

```diff
@@ -76,9 +76,9 @@
       "integrity": "sha1-DNkKVhCT810KmSVsIrcGlDP60Rc=",
       "dev": true,
       "requires": {
-        "kind-of": "3.2.2",
-        "longest": "1.0.1",
-        "repeat-string": "1.6.1"
+        "kind-of": "^3.0.2",
+        "longest": "^1.0.1",
+        "repeat-string": "^1.5.2"
```

Comment:
some deps require an older version now... is this intentional?

Reply:
Looks strange at first sight, but it seems that npm has switched from exact version numbering to caret version ranges for the `requires` property. As far as I can say, this does not affect direct dependend packages (those explicitly stated in package.json). Otherwise the package-lock.json would be somehow useless.

So, in the above example `"repeat-string": "^1.5.2"` will also include `"repeat-string": "1.6.1"` automatically.

Reply:
According to some search results, it seems that this change was introduced with version 6 of npm.